### PR TITLE
Add polarion test case and test run scripts into robottelo

### PR DESCRIPTION
**Pull request by lpramuk**
- added polarion test case upload script
- added polarion test run upload script

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # Version updates managed by pyup.io
 
 attrdict==2.0.1
+betelgeuse==1.7.0
```
Comment (mshriver) [inline requirements.txt:5]: I think this should go into `requirements-optional.txt` (Ideally we refactor anyway to use setuptools extra_requires groups),  as its not directly related to actually executing pytest from robottelo.

File: scripts/polarion-test-case-upload.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# requires betelgeuse pypi package
```
Comment (mshriver) [inline scripts/polarion-test-case-upload.sh:1]: Please provide more thorough docblocks for both scripts explaining their use, arguments, defaults, etc.

File: scripts/polarion-test-case-upload.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+TC_PATH='tests/foreman/api tests/foreman/cli tests/foreman/ui tests/foreman/longrun tests/foreman/sys tests/foreman/installer tests/foreman/rhai tests/foreman/endtoend tests/foreman/virtwho'
```
Comment (mshriver) [inline scripts/polarion-test-case-upload.sh:3]: Can we instead exclude the paths this should not be run against, or at least comment what we're avoiding here? Looks like all of `tests/foreman` could be included.

File: scripts/polarion-test-run-upload.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    --response-property "${POLARION_SELECTOR}" \
+    --test-run-title "${TEST_RUN_ID}" \
+    --test-run-id "${TEST_RUN_ID}" \
+    --test-run-group-id "${TEST_RUN_GROUP_ID}" \
+    --status inprogress \
+    "./${ENDPOINT}-results.xml" \
+    tests/foreman \
+    "${POLARION_USERNAME}" \
+    "${POLARION_PROJECT}" \
+    "sat-${IMPORTANCE}-results.xml"
```
Comment (mshriver) [inline scripts/polarion-test-run-upload.sh:24]: I think this (and the ENDPOINT-results.xml on L20) needs to be more configurable. Its relevant to how we're writing CI, but these envvars won't be relevant or set for component owners trying to upload results.

Comment (lpramuk) [reply scripts/polarion-test-run-upload.sh:24]: oops I think I already amended the line, watch me repushing PR

File: scripts/polarion-test-case-upload.sh (hunk 3 above)
Comment (lpramuk) [reply scripts/polarion-test-case-upload.sh]: Refactored.
it has no sense

**Comment by omkarkhatavkar** [comment]
Can one of the admins verify this patch?

File: requirements.txt (hunk 1 above)
Comment (lpramuk) [reply requirements.txt]: removing from `requirements.txt`

**Review by mshriver — APPROVED** [review]
Approving because the addition of these scripts don't impact the framework itself, but with some comments.

1. This should be implemented as a robottelo-cli command group in `Click`
2. The betelgeuse requirement should be included in requirements-optional.txt, as its required to run this script.

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**